example/lib/main.dart

```diff
@@ -14,6 +16,12 @@ class MyApp extends StatelessWidget {
       ),
       home: Scaffold(
         body: ShowCaseWidget(
+          onStart: (index, key) => {
```

Comment:
This contains incorrect lambda structure. `=> {}` will create a set. Either use only `=>` or  only `{}`.